build_tools/circle/build_test_arm.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+
+set -e
+set -x
+
+UNAMESTR=`uname`
+
+setup_ccache() {
+    echo "Setting up ccache"
+    mkdir /tmp/ccache/
+    which ccache
+    for name in gcc g++ cc c++ x86_64-linux-gnu-gcc x86_64-linux-gnu-c++; do
+      ln -s $(which ccache) "/tmp/ccache/${name}"
+    done
+    export PATH="/tmp/ccache/:${PATH}"
+    ccache -M 256M
+}
+
+# imports get_dep
+source build_tools/shared.sh
+
+sudo add-apt-repository --remove ppa:ubuntu-toolchain-r/test
+sudo apt-get update
+sudo apt-get install python3-scipy python3-matplotlib libopenblas-base \
+                     python3-virtualenv ccache
```

Comment:
Since we are using a virtualenv, there is no need to install python packages such as scipy, matplotlib, and openblas from the system packages as they will be re-installed later when building scikit-learn.

```suggestion
sudo apt-get install python3-virtualenv ccache
```